.github/workflows/add-to-projects.yml

```diff
@@ -99,9 +99,21 @@ jobs:
           github-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
           # any of the following labels:
           labeled: component/clients, component/spring-sdk, component/camunda-process-test, component/c8-api
+      - name: Checkout team-qa-engineering repo
+        uses: actions/checkout@v5
+        with:
+          repository: camunda/team-qa-engineering
+          path: team-qa-engineering
       - id: add-to-qualityboard
         name: Add Bugs to Quality Board project
-        uses: camunda/team-qa-engineering/automation/actions/add-to-quality-board@v1
+        uses: ./team-qa-engineering/automation/actions/add-to-quality-board
```

Comment:
can we instead move this action to https://github.com/camunda/camunda/tree/main/.github/actions?

I don't think using private repo source (for a public repo itself) here is a good pattern. It also makes this action harder to use.

Alternartively, you can move your action in an existing public repo like https://github.com/camunda/infra-global-github-actions.